modules/apps/map/map-common/test/liferay/MapBase.js

```diff
@@ -162,6 +150,8 @@ describe('MapBase', () => {
 		it('disposes existing search custom control', () => {
 			const control = {dispose: jest.fn()};
 
+			mapImpl._geoJSONLayer = {dispose: jest.fn()};
```

Comment:
In this situation (without the added line) we get the following error `TypeError: this._geoJSONLayer.dispose is not a function`. I've looked into the implementation of the geoJSONLayer and how it's used in the test above and concluded that the error is correct and simply added a mock `dispose` function to it.